test/spec/cloud-element-templates/cmd/ChangeElementTemplateHandler.spec.js

```diff
@@ -372,8 +374,7 @@ describe('cloud-element-templates/cmd - ChangeElementTemplateHandler', function(
 
           const taskDefinition = findExtension(task, 'zeebe:TaskDefinition');
 
-          expect(taskDefinition).to.exist;
```

Comment:
Let's clearly document this change in behavior. I believe we agreed this is a bug fix?